docs/api/admin/feature-toggles-api.md

```diff
@@ -144,6 +153,46 @@ Used by the admin dashboard to update a feature toggles. The name has to match a
 
 Returns 200-respose if the feature toggle was updated successfully.
 
+### Tag a Feature Toggle
+
+`POST https://unleash.host.com/api/admin/features/:featureName/tags`
+
+Used to tag a feature
+
+If the tuple (type, value) does not already exist, it will be added to the list of tags. Then Unleash will add a relation between the feature name and the tagId
```

Comment:
Will the type similarly be added implicitly?

Reply:
I think that makes sense for simplicity. This also requires us to be careful and use "upsert" when we have plugins registering tag-types to make sure they are properly defined.